package.json

```diff
@@ -34,7 +34,8 @@
     "./lib/simple-auth.js": false
   },
   "dependencies": {
-    "websocket": "^1.0.24"
+    "websocket": "^1.0.24",
+    "metarhia-common": "^0.0.9"
```

Comment:
This package won't be installed at all because npm will just use `package-lock.json` if it is present (and it is). Please make sure that you use npm5.